docs/one-click-integrations/bitbucket-integration.md

```diff
@@ -0,0 +1,170 @@
+# Bitbucket integration
+
+!!! note
+    (CUSTOMER FACING MESSAGE NEEDED) We have launched Bitbucket integration to receive your feedback. Some metrics are not available yet.
+
+Pulse integrates with Bitbucket Cloud to receive data about changes and deployments, necessary to calculate the metrics:
+
+-   [Lead time for changes](../metrics/accelerate.md#lead-time-for-changes)
```

Comment:
For V1, we’ll only calculate the following accelerate metrics in real time:

- Deployment frequency
- Lead time for changes

All the remaining metrics will be delivered for V3:

- Changes metrics (Time to open, Time to merge)
-  Reviews metrics (Time to first review, Time to first approval, Time from first to last approval, Time from last approval to merge, Outlier review details)
- Work in progress

@mrfyda Can you please confirm?

Reply:
that's right clau! :)